system_tests/bigquery.py

```diff
@@ -188,7 +195,11 @@ def test_patch_table(self):
     def test_update_table(self):
         dataset = Config.CLIENT.dataset(DATASET_NAME)
         self.assertFalse(dataset.exists())
-        dataset.create()
+
+        @retry(Forbidden, tries=3, delay=20)
+        def create_dataset():
+            dataset.create()
+        create_dataset()
```

Comment:
I think pylint will want an empty line between the fn. definition and the next line of code.